homeassistant/components/rainbird/quality_scale.yaml

```diff
@@ -43,16 +43,22 @@ rules:
       needs to be moved to `runtime_data`.
 
```

Comment:
General note, the `_async_fix_entity_unique_id` and `_async_fix_device_id` both have `hass` as parameter but that is unused